src/apple/project.rs

```diff
@@ -121,5 +135,17 @@ pub fn gen(
         .with_arg(dest.join("project.yml"))
         .run_and_wait()
         .map_err(Error::XcodegenFailed)?;
+
+    if !ios_pods.is_empty() || !macos_pods.is_empty() {
+        std::env::set_current_dir(&dest)
```

Comment:
It looks like `pod install` lets you specify a project dir, if that would be equivalent: https://guides.cocoapods.org/terminal/commands.html#pod_install